gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -5985,6 +5985,8 @@ components:
                 - UNPUBLISH_API
                 - UNPUBLISH_API_RESULT
                 - UNPUBLISH_DICTIONARY
+                - START_ENVIRONMENT_FLOW
+                - STOP_ENVIRONMENT_FLOW
```

Comment:
same just a preference: 😁 
I'd use the same "jargon" as the product. if it's called a deploy and undeploy, I'd add it here. 
start stop makes it look like there's something spinning in the background 🤷‍♂️

Reply:
renamed to DEPLOY_ENVIRONMENT_FLOW